modules/openapi-generator/src/main/resources/python-flask/base_model_.mustache

```diff
@@ -22,34 +22,37 @@ class Model(object):
     attribute_map = {}
 
     @classmethod
-    def from_dict(cls{{^supportPython2}}: typing.Type[T]{{/supportPython2}}, dikt){{^supportPython2}} -> T{{/supportPython2}}:
+    def from_dict(cls: typing.Type[T], dikt, json_keys=True) -> T:
         """Returns the dict as a model"""
-        return util.deserialize_model(dikt, cls)
+        return util.deserialize_model(dikt, cls, json_keys=True)
 
-    def to_dict(self):
+    def to_dict(self, json_keys=True):
         """Returns the model properties as a dict
-
+        
+        :param json_keys: Defines if json_keys are used in dict
+        :type: bool
         :rtype: dict
         """
         result = {}
 
-        for attr, _ in six.iteritems(self.openapi_types):
+        for attr in self.openapi_types:
             value = getattr(self, attr)
+            dict_attr = self.attribute_map[attr] if json_keys else attr
```

Comment:
Why not reassign the value of attr here?
```
if if json_keys:
    attr = self.attribute_map[attr]
```
We don't use it anywhere else below.